### ISSUE TITLE
HACKING refers to missing script contrib/uncrustify.sh

### ISSUE THREAD

**Issue by superm1** [gicmo/bolt#45]
```
Run contrib/uncrustify.sh to format the source code before submitting pull requests.
```
That script doesn't exist though in master.

**Comment by gicmo** [comment]
Thanks for spotting. It moved to `scripts`. Will fix tomorrow.

**Comment by gicmo** [comment]
Fixed with 55bb74ce8b5c96192b250b6b1b6d6fad70b349de
